src/util/tracking.js

```diff
@@ -86,11 +86,23 @@ async function logToFirebase(event: TrackingEvent, values: TrackingValues) {
   }
 }
 
+export async function multicastUtilServers(path: string, options?: Object): Promise<any> {
+  const serverUrls = ['https://util2.edge.app/', 'https://util1.edge.app/']
+  let reply
+  for (const url of serverUrls) {
+    reply = fetch(`${url}${path}`, options)
```

Comment:
This should use asyncwaterfall as you do not always want to talk to both servers otherwise you will cause duplicate entries. Put in an eight second time out between each server call. That should just be a parameter to asyncwaterfall